content/posts/0006-build-plan-buildpack.md

```diff
@@ -0,0 +1,93 @@
+---
+title: What is the Build Plan Buildpack and why should I care?
```

Comment:
Just a suggestion: maybe think of a more catchy title. One example "The power of the Build Plan Buildpack"